README.md

```diff
@@ -14,3 +14,8 @@ part of the array each recursive call considers.
 Analyse the time complexity of your implementation and give a $\Theta$ bound for
 its worst-case runtime. Add your answer, including your reasoning, to this
 markdown file.
+
+Complexity Analysis: 
+The runtime for this version of mergesort without the recursive stuff. The runtime is practically the same as the original because it is doing the same thing as the normal mergesort. Now if we were to divide it into bigger 
+subarrays then that would change the runtime analysis. In the end, the runtime analysis should be $\Theta$ (nlogn) is what the runtime is. With divide and conquer that is going to be different because that algorithm is 
+dividing everything into three subarrays. This mergesort is the same as it is listed in the slides provided to us by Professor Lars. 
```

Comment:
Aren't you doing extra work to keep it in place?

Reply:
Yes, you are doing more work but with the nonrecursive side of things it will reduce the amount of times you have to call the recursive loops and that can reduce the overload that they might occur. I thought they would have the same run time because they are practically doing the same thing except with recursive you have the loops which might get overloaded with information where the non-recursive just does it one at a time.

Reply:
It might help to analyze just the merge step.

Reply:
My version of the merge step starts with an array of one and doubles until it gets to array size and then it starts to divide. Oh wait so pretty much both recursive and non-recursive are doing the same thing but my recursive is doubling the array. Which is uses loops to divide the subarrays. Whereas the recursive function which has something to handle the divisions. So the non-recurisve one is just theta(N) correct?

Reply:
sorry I thought about this more so I would have to square the n at the start of the (nlogn) because it is acting as exponential at the merge step so it would be theta(n^2logn)

Reply:
That sounds like it's going in the right direction.

Reply:
I just updated the Readme file. I tried to explain everything that I wrote on my iPad in text.